content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -0,0 +1,59 @@
+# Add a model
+
+You can add models from any ML project to the model registry. To add a model to
+your model registry, Iterative Studio creates an annotation for it in an
+`artifacts.yaml` file in your Git repository. If you are using the GTO command
+line tool, you can also add models from the [CLI][gto]. To add models using
+Iterative Studio, watch this tutorial video or read on below:
+
+https://youtu.be/szzv4ZXmYAs
```

Comment:
- [ ] Make video public (and I think the link should be https://www.youtube.com/watch?v=szzv4ZXmYAs)

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/1477535/180920139-7cb96367-9385-4b20-b25a-043cdbc07628.png">